service/src/test/java/bio/terra/catalog/service/DatasetServiceTest.java

```diff
@@ -54,14 +59,30 @@ class DatasetServiceTest {
   private final AuthenticatedUserRequest user = mock(AuthenticatedUserRequest.class);
   private final DatasetId datasetId = new DatasetId(UUID.randomUUID());
   private final String sourceId = "sourceId";
+  private final String workspaceSourceId = "abc-def-workspace-id";
   private final String metadata = """
         {"name":"name"}""";
   private final Dataset dataset =
       new Dataset(datasetId, sourceId, StorageSystem.EXTERNAL, metadata, null);
+  private final Dataset tdrDataset =
+      new Dataset(
+          new DatasetId(UUID.randomUUID()),
+          sourceId,
+          StorageSystem.TERRA_DATA_REPO,
+          metadata,
+          null);
+  private final Dataset workspaceDataset =
+      new Dataset(
+          new DatasetId(UUID.randomUUID()),
+          workspaceSourceId,
+          StorageSystem.TERRA_WORKSPACE,
+          metadata,
+          null);
```

Comment:
I think we have enough slight differences between our calls that I would rather not make a helper. I see the value, but I worry introducing the intermediate name would confuse things more. I could see a "createDefaultDataset" helper, but I wouldn't want to pick arbitrary fields to be different, because then we start getting weird overlap. This gives us some flexibility to make the metadata different if we needed.

(We also have three that are identical except for metadata used in the tests themselves)